### PR TITLE
Added argparse/optparse for cmdline options to pyjscompressor.py

### PR DESCRIPTION
**Pull request by happyalu**

### CODE REVIEW

File: contrib/pyjscompressor.py [hunk 1]
```diff
@@ -267,25 +272,88 @@ def compress_all(path):
...
+            print("Could not determine CPU Count. Using One process")
+            num_procs = 1
+
+    print("Running %d processes" % num_procs)
+
+try:
+    compress_all(directory)
+except KeyboardInterrupt:
+    print('')
+    print('Compression Aborted')
```
Comment (anthonyrisinger) [inline contrib/pyjscompressor.py:359]: this try block needs to be indented and inside of `if __name__ == __main__`

**Comment by anthonyrisinger** [comment]
@happyalu, this all looks good, but see my comment -- the last try block is not inside `if __name__ == __main__:` check.

... once that's fixed should be no issue to pull.

**Comment by anthonyrisinger** [comment]
actually i'll just fix it :-) thanks @happyalu!

**Comment by happyalu** [comment]
@xtfxme Thanks a lot!

**Pull request merged**